Platform/Udoo/UdooNeo_iMX6SX_1GB/Drivers/GraphicsOutputDxe/GraphicsOutputDxe.c

```diff
@@ -360,92 +297,3 @@ GraphicsOutputDxeInitialize(
     return status;
 }
 
-// -------------------------------------------------------------------------------------------------------
-
-// Almost Identical to iMXVideoDxe.c VidGopBlt

```

Comment:
Almost identical.  Any idea what the difference was?

Reply:
Only difference is the frameBuffer/frameWidth assignment to use its internal global structure. In the library implementation, I pull the frame info from the *This argument